x-pack/legacy/plugins/integrations_manager/server/saved_objects.ts

```diff
@@ -11,11 +12,12 @@ export const getClient = (req: Request) => req.getSavedObjectsClient();
 export const mappings = {
   [SAVED_OBJECT_TYPE]: {
     properties: {
-      is_working: {
-        type: 'boolean',
-      },
-      other: {
-        type: 'keyword',
+      installed: {
+        type: 'nested',
+        properties: {
+          id: { type: 'keyword' },
+          type: { type: 'keyword' },
```

Comment:
I think we'll probably need a title and/or a description here for the UI eventually, but we can punt for now.